invenio_assets/ext.py

```diff
@@ -71,6 +71,10 @@ def init_config(self, app):
             'COLLECT_FILTER', partial(collect_staticroot_removal, app))
         app.config.setdefault(
             'WEBPACKEXT_PROJECT', 'invenio_assets.webpack:project')
+        if app.debug:  # for development use 2-level deep symlinking
+            from pywebpack.storage import LinkStorage
+            app.config.setdefault(
+                'WEBPACKEXT_STORAGE_CLS', partial(LinkStorage, depth=2))
```

Comment:
I am just thinking should `depth` be configurable?

Reply:
I didn't go with introducing another config variable since `invenio-assets` doesn't actually have a `config.py`. 

Another reason is that it's still possible to import `LinkStorage` in your config and customize via `partial`, and in case we add more parameters (e.g. to force converting files to symlinks), we would still be flexible. An alternative is going all-in with the classic 'function-as-import-string"

Reply:
No, I understood now that the depth 2 goes well along with the folder convention `/js/<module-name>` so I think is fine there ;)

Reply:
Currently there's https://github.com/inveniosoftware/invenio-previewer/issues/108 pending to have the naming convention consistent throughout all Invenio modules